src/Acuminator/Acuminator.Analyzers/StaticAnalysis/SavingChanges/SaveOperationHelper.cs

```diff
@@ -61,6 +62,28 @@ internal static class SaveOperationHelper
 			return SaveOperationKind.None;
 		}
 
+		public static PXDatabaseKind GetPXDatabaseSaveOperationKind(IMethodSymbol symbol, PXContext pxContext)
+		{
+			symbol.ThrowOnNull(nameof(symbol));
+			pxContext.ThrowOnNull(nameof(pxContext));
+
+			var containingType = symbol.ContainingType?.OriginalDefinition;
+
+			if (containingType != null && 
+			    containingType.InheritsFromOrEquals(pxContext.PXDatabase.Type))
+			{
+				if (string.Equals(symbol.Name, DelegateNames.Insert))
```

Comment:
Here is actually a good place for switch

Reply:
Operator `switch` works with constants in cases. There are `static readonly` variables "DelegateNames..."

Reply:
I see. You coud have imitate switch with dictionary but I think it's not worth the effort. Let's leave it like this then